code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -456,6 +514,7 @@
 ///Nulls the target of our hunter's mark
 /datum/action/ability/activable/xeno/hunter_mark/proc/unset_target()
 	SIGNAL_HANDLER
+	to_chat(owner, span_xenodanger("We cannot maintain our focus on [marked_target] any longer."))
```

Comment:
This should be a balloon alert

Reply:
Good idea though I should shorten it in that case.